dateutil/tz/tz.py

```diff
@@ -1460,6 +1458,50 @@ def __get_gettz():
         tzlocal_classes += (tzwinlocal,)
 
     class GettzFunc(object):
+        """gettz(name=None)
+
+        Given a variety of inputs, like time zone name and ``TZ`` varible,
+        return the tz (aka Olson) database time zone.
+
+        :param name:
+            A time zone name, a ``TZ`` variable, or ``None``.
+
+        :return:
+            A :class:`dateutil.tz.tzfile`. If name is ``None``,
```

Comment:
This is also not accurate. If you look at the code, it can return `tzfile`, `tzlocal`, `tzstr`, possibly others.

This function is that it takes a string representing a time zone and returns the best-matching `tzinfo` subclass instance. The specific subclass returned is not part of the function's contract.